Cubical/Core/Prelude.agda

```diff
@@ -45,6 +45,10 @@ cong : ∀ {B : A → Set ℓ'} (f : (a : A) → B a) (p : x ≡ y)
        → PathP (λ i → B (p i)) (f x) (f y)
 cong f p = λ i → f (p i)
 
+cong′ : ∀ {B : Set ℓ'} (f : A → B) {x y : A} (p : x ≡ y)
+      → Path B (f x) (f y)
+cong′ f p = λ i → f (p i)
```

Comment:
Why is this here? Can't you just use `cong`?

Reply:
The fact that `B` is guaranteed to not be dependent makes constraint solving easier.
Sometimes if you use `cong` instead you get unsolved metas.

That's why we have a lot of primed combinators in [`Function`](https://agda.github.io/agda-stdlib/Function.html) in the stdlib.

Reply:
Oh I see! I guess we should have this as well then. Thanks for clarifying.

Reply:
Actually, @ice1000 can you please add a comment explaining this? Otherwise I will surely forget and delete this one day...

Reply:
Yes I can.